manual/source/Tutorial/GettingStarted/ParsingCommandLineArguments.rst

```diff
@@ -639,3 +594,35 @@ Below, you can see a part of the rendered HTML and man pages generated by the co
 .. image:: modify_string.png
 
 For further deading, have a look at the :dox:`ArgumentParser` class.
+
+The Version Update Feature
+--------------------------
+
+With the seqan-2.3.0 release applications, using the :dox:`ArgumentParser`, check SeqAn servers for version updates. The functionality helps getting new versions out to users faster. It is also used to inform application developers of new versions of the SeqAn library which means that applications ship with less bugs.
+
+For example, you might get a message like this:
+
+.. code-block:: console
+
+  [APP INFO] :: There is a newer version of this application available.
+  [APP INFO] :: If this app is developed by SeqAn, visit www.seqan.de for updates.
+  [APP INFO] :: If you don't want to recieve this message again set --version_check OFF\n\n";
+
+The process of checking for a new version happens at most once a day. During this process the following information is transmitted to the servers:
+
+  * The application name and its version
+  * The SeqAn version that the application was built with
+  * The operating system type (Linux, macOS, Windows or BSD)
+  * The CPU type (32 or 64bit)
+
+The server counts the total number of version requests and may also resolve them to geographical regions, however IP-Addresses are not stored permanently and there is no form of user identification and no tracking. SeqAn collects nothing related to your use of the application like selected options and of course also nothing regarding your files!
+
+.. attention::
+  SeqAn does not collect any personal data. The collection of technical information as stated above is used exclusively for internal feedback and statistical analysis and never shared nor sold to third-party companies. 
+
+If you still feel uncomfortable with the version check, you may
```

Comment:
I think it's sufficient to note this in the paragraph above.

If I include this here again, I think it sounds like you should turned it off **in general** if you think you might at some point measure performance?

" If you don't feel comfortable or you want to run some performance measurements, you may ... disable this feature at compile time...."

?